dlhub_sdk/models/servables/python.py

```diff
@@ -153,8 +155,8 @@ class PythonStaticMethodModel(BasePythonServableModel):
     """
 
     @classmethod
-    def create_model(cls, module, method, autobatch=False, function_kwargs=None):
-        """Initialize the method
+    def create_model(cls, module=None, method=None, autobatch=False, function_kwargs=None, *, f=None):
+        """Initialize the method based on the provided arguments
```

Comment:
what's the purpose of allowing the user to either pass in `f` or the module and method name directly?

Reply:
just gives the user more options. no functionality is added here (in terms of the entire codebase) other than allowing the user to do it (or really, `easy_publish`) from `create_model`.